ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -458,60 +451,56 @@ def check_backends(logger):
         """
 
         logger.info("Checking available ALBA backends ...")
+        if System.get_my_storagerouter().node_type == 'EXTRA':
+            return logger.skip("Skipping ASD check because this is a EXTRA node ...")
```

Comment:
Why skip this check for EXTRA nodes? EXTRA nodes can also have ASDs to be checked no?
Also i don't think this should be skipped for EXTRA nodes, but this should only be executed on 1 node (any node)